docs/PacketFence_Aerohive_Quick_Install_Guide.asciidoc

```diff
@@ -18,433 +18,267 @@ include::includes/global-attributes.asciidoc[]
 
 About this Guide
 ----------------
-This guide has been created in order to help sales engineers, product managers, or network specialists demonstrate the PacketFence capabilities on-site with an existing or potential customer.  It can also provide guidelines to setup a proof of concept for a potential PacketFence deployment using the Aerohive equipment.
+This guide has been created in order to provide guidelines to deploye a Aeroive Wireless APs and the associated management in a PacketFence contexte, wether it is with a standalown server, a cluster or PacketFencZEN deployement.
 
-The instructions are based on version {release_version} of the PacketFence Live Image.
+NOTE: AeroHIVE products are a bit different compared to the other vendors. They support either a local HiveManager (kind of wireless controller) or a cloud-based HVM. However, the configuration is the same for the local and the cloud-based controller. Note that all the config are made on the HVM and then pushed to the APs. The instructions are based on version {release_version} of the PacketFence.
 
 Assumptions
 -----------
-* You will need a USB key with a capacity of at least 4GB;
-* You will need a laptop or a PC from which the PacketFence Live USB key will run from;
-* You will deploy a VLAN enforcement environment according to the customer specifications;
-* Your customer will provide the network equipment (switch, cables, etc) to interconnect this PoC setup;
+* You already have a PacketFence server(s) up and running, (Standalone, cluster or ZEN)
+  ** If not, please refer to the https://packetfence.org/doc/PacketFence_Administration_Guide.html[PacketFence Administration Guide].
+* The network architecture is in order to give acces to the Aerohive Access Point, and have access to internet.
+* The VLANs are defined for registration, isolation and management networks.
+ * From this doc, we will assume that the VLANs tags are define like following :
+  ** Registration VLAN ID 2, Subnet 192.168.2.0/24,
+  ** Isolation VLAN ID 3, Subnet 192.168.3.0/24,
+  ** Production VLAN ID 10, Subnet 172.16.1.0/24,
+* The VLans are spanned in the swtichs and switching L2 equipments, from the _Production Network_ to the PacketFenceServer(s)
+* The VLANs are allowed in the trunks.
 * Aerohive Access Point is loaded with HiveOS with version 6 or later;
 * HiveManager with version 6 or later.
 
+'''
```

Comment:
?

Reply:
This is the code for a horizontal rule in asciidoc.
http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#horizontal-rules-and-page-breaks